UndertaleModTool/Scripts/Community Scripts/FixAlphysLabCrashAndroid.csx

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Linq;
+using UndertaleModLib.Util;
+
+EnsureDataLoaded();
+
+if (Data?.GeneralInfo?.DisplayName?.Content.ToLower() == "undertale")
+{
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Create_0", "glowamt = 0.5", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Create_0", "if (global.plot > 125)", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Create_0", "if (scr_murderlv() >= 12)", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Create_0", "instance_destroy()", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Draw_0", "x1 = (obj_mainchara.x - 10)", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Draw_0", "x2 = (obj_mainchara.x + 30)", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Draw_0", "y1 = (obj_mainchara.y - 5)", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Draw_0", "y2 = (obj_mainchara.y + 35)", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Draw_0", "draw_sprite_ext(spr_darkhalo_big, 0, x1, y1, 1, 1, 0, c_white, glowamt)", "");
+    ReplaceTextInGML("gml_Object_obj_labdarkness_Draw_0", "draw_set_alpha(1)", "");
+    ReplaceTextInGML("gml_Object_obj_labcamera_Create_0", "if (global.osflavor == 2)", "if (global.osflavor == 4)");
```

Comment:
It might affect it, but this script is only meant to be used for android, not for windows or desktop generally.

Reply:
Hm, maybe i got a different impression then. My impression was that it was intended as a script so that you can convert your current ut mod to be used on android. and in that case, since people will likely still work on their mod, it would be useful if desktop would thus not regress.

Reply:
Using the same mod file for android on the desktop game isn't really a good idea, especially that the user will have touch controls on his window although he'll be using keyboard. I think mod developers would just patch mods on a copy of the data.win, not the one being used in development. 

I dunno, tell me your suggestions.

Reply:
Oh, somehow it completely slipped that the touch controls will always be displayed.

IMO having one shared codebase is good, but for this script it probably is a bit too out of scope to make it compatible with both desktop and android, so feel free to just keep as is then.